Etiquette/CODE_REVIEW.md

```diff
@@ -0,0 +1,108 @@
+* If you see a stale pull request without any updates - ask for the status update in the comment
+* Do not be a blocker - if you requested changes and they were addressed make sure you review it
+* Do not merge others pull requests - they may be waiting for some one else review
+
+Responding to feedback
+-------------------------
+
+* Wait before answering to the single comments, reviewer may have more (though they should prefer to send all comments at once)
+* Be grateful for the reviewer's suggestions. ("Good call. I'll make that
+  change.")
+* If you disagree with suggestion explain way and avoid template answers like "out of the scope", "thank you for your opinion". If you think that suggestion is a personal preference and does not bring value - say so
+* Assume the best intention from the reviewer's comments.
+* If a reviewer seems aggressive or angry or otherwise personal, consider if it is intended to be read that way and ask the person for clarification of intent, in person if possible. Do not escalate online discussions.
+* If you can't reach agreement seek for more opinions
+* Don't be too defensive about your code, someone will rewrite it anyway sooner or later (it might be yourself as well)
+* Try to answer all comments. You can review your own code for that instead of sending individual replies
+* Add "work in progress" label while addressing review comments
+* Re-request review when you addressed the comments (either implementing suggested change or replying to the comment) and label it with "Ready for review" again
+* Do not rebase and force-push in your pull request branch after you recieved any comments, use merge intsead. All pull requests are squashed when they are merged into the base branch so the history of your changes won't matter. It's fine to rebase and force-push if there are no comments
+* Do no merge pull request with pending requests for changes unless you agreed with their authors that it was addressed
+* When you have two approvals and don't wait for review from anyone else - put a "Merge" label for bot to merge your changes.
+
+
+PR Parties
+--------
+
+To improve colloboration every week (on Monday) we take time to review pull request together in small groups, what we call "PR parties"
+
+- Each group should include at least one senior developer. 
+- Each group will have few pull requests assigned to them. 
+- When done with assigned PRs group is free to review something else (i.e. each other pull requests) or finish the meeting
+
+Pull Assigners
+--------
+
+To balance the workload and avoid knowledge silos we use Pull Assigners to assign reviewers.
+
+- Each pull request will be assigned three random reviewers from the team
+- Two approvals are enough to merge pull request (there shouldn't be pending requests for changes though)
+- Anyone else is still welcome to review any pull request and author can request for reviews from specific team mates
```

Comment:
Something I think we should clarify is "protocol" for when your PR has two or more reviewers that are absent and cannot review the PR. I am unsure if this would be a technical solution (There's a small discussion about this in https://github.com/pullreminders/backlog/issues/158) or if it's an agreement that we re-request reviews if there are not 2 or more eligible reviewers requested.

Reply:
I think this will need a wider discussion with the team, so does not have to. be changed in this PR